test/js/node/stream/node-stream.test.js

```diff
@@ -386,6 +386,27 @@ it("Readable.fromWeb", async () => {
   expect(Buffer.concat(chunks).toString()).toBe("Hello World!\n");
 });
 
+it("Writable.toWeb", async () => {
+  let dataBuffer = "";
+
+  // Create a Node.js Writable stream
+  const nodeWritable = new Writable({
+    write(chunk, encoding, callback) {
+      dataBuffer += chunk.toString();
+      callback();
+    },
+  });
+
+  const webWritable = Writable.toWeb(nodeWritable);
```

Comment:
This should have tests that verify it throws an appropriate exception when invalid input is passed

Reply:
tested